src/Xamarin.Android.Build.Tasks/Tasks/ConvertResourcesCases.cs

```diff
@@ -54,7 +54,7 @@ void FixupResources (ITaskItem item, Dictionary<string, string> acwMap)
 			foreach (string file in xmls) {
 				Log.LogDebugMessage ("  Processing: {0}", file);
 				var srcmodifiedDate = File.GetLastWriteTimeUtc (file);
-				var tmpdest = file + ".tmp";
+				var tmpdest = file + "_" + Path.GetFileName (Path.GetTempFileName ());
```

Comment:
Fair idea, not-quite-write implementation.

Did you know that [`Path.GetTempFileName()`](https://msdn.microsoft.com/en-us/library/system.io.path.gettempfilename(v=vs.110).aspx) *creates the file*?

> This method creates a temporary file with a .TMP file extension. 

For example:

```csharp
var p = Path.GetTempFileName();
// p == e.g. "/var/folders/1y/wwmg3hv5685ft661f5q2w2100000gn/T/tmpa1d2e05.tmp"
File.Exists(p); // is true
```

As such, `Path.GetFileName(Path.GetTempFileName())` will "litter" the user's `%TMPDIR%` directory with temporary files which are never deleted.

*Instead*, we should just do:

```csharp
var tempest = Path.GetTempFileName ();
```

We'll properly delete this file in the `finally` block below.